media-libs/opencollada/opencollada-9999.ebuild

```diff
@@ -1,92 +1,70 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2016 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
 
-if [[ ${PV} == *9999* ]] ; then
-	SCM_ECLASS="git-r3"
-else
-	SCM_ECLASS="vcs-snapshot"
-fi
-
-inherit versionator eutils multilib cmake-utils ${SCM_ECLASS}
+inherit multilib cmake-utils git-r3
 
 DESCRIPTION="Stream based read/write library for COLLADA files"
 HOMEPAGE="http://www.opencollada.org/"
 LICENSE="MIT"
-IUSE="expat"
+IUSE="expat static-libs"
 
 # seems like the Khronos Group hasnt invented the SOVERSION yet
-MY_SOVERSION="1.2"
+MY_SOVERSION="1.6"
 
 SLOT="0"
 
-if [[ ${PV} != *9999* ]]; then
```

Comment:
You could keep these lines and update 9999 with the fixed SRC_URI so that future releases can be easily created from live ebuild.